src/v2/Apps/Feature/Components/FeatureHeader.tsx

```diff
@@ -62,17 +62,17 @@ export const FeatureHeader: React.FC<FeatureHeaderProps> = ({
 
       <Meta p={4} flexBasis={image ? "50%" : "100%"}>
         <Join separator={<Spacer my={1} />}>
-          <Sans size="10" element="h1" textAlign="center" unstable_trackIn>
+          <Text
+            variant="largeTitle"
+            as="h1"
+            fontSize="size10"
```

Comment:
So this guy steps out of the codified variants a bit. There are still tokenized sizes available: https://github.com/artsy/palette/blob/271545bc359f426f3d82dc65c2bda02aa6b1367f/packages/palette/src/elements/Text/tokens.shared.ts#L5-L16 and this utilizes one since it needs to be a bit larger than `largeTitle`. I used it since we still want the `largeTitle` other styles: `lineHeight: title` & `letterSpacing: tight`.